packages/common/src/core/schemas/internal/groupsToComboBoxItems.ts

```diff
@@ -0,0 +1,32 @@
+import { IGroup } from "@esri/arcgis-rest-portal";
+import { IUiSchemaComboboxItem } from "../types";
+
+/**
+ * Convert an array of groups into a UiSchemaComboboxItem array
+ */
+export function groupsToComboBoxItems(
```

Comment:
minor, but I feel like this function name is a little misleading since we're not _just_ converting groups to combobox items, we're also performing a "filter" of sorts to only include groups a user can share to

Reply:
100% agree. Will rename to `getSharableGroupsComboBoxItems`